modules/web-mercator/docs/api-reference/web-mercator-viewport.md

```diff
@@ -1,38 +1,35 @@
 # WebMercatorViewport
 
-The main purpose of the `PerspectiveMercatorViewport` is to enable 3D rendering to
-seamlessly overlay on top of map components that take web mercator style
-map coordinates (`lat`, `lon`, `zoom`, `pitch`, `bearing` etc),
-and to facilite the necessary mercator projections by breaking them into a
-minimal non-linear piece followed by a standard projection chain.
+The `WebMercatorViewport` class takes map camera states (`latitude`, `longitude`, `zoom`, `pitch`, `bearing` etc.),
+and performs projections between world and screen coordinates.
 
 
-## Methods
-
-### Constructor
+## Constructor
 
 | Parameter     |  Type    | Default | Description                                                |
 | ------------- | -------- | ------- | ---------------------------------------------------------- |
-| `width`       | `Number` | `1`       | Width of "viewport" or window                              |
-| `height`      | `Number` | `1`       | Height of "viewport" or window                             |
-| `latitude`    | `Number` | `37`      | Center of viewport on map (alternative to center)          |
-| `longitude`   | `Number` | `-122`    | Center of viewport on map (alternative to center)          |
-| `zoom`        | `Number` | `11`      | Scale = Math.pow(2,zoom) on map (alternative to opt.scale) |
-| `pitch`       | `Number` | `0`       | Camera angle in degrees (0 is straight down)               |
-| `bearing`     | `Number` | `0`       | Map rotation in degrees (0 means north is up)              |
-| `altitude`    | `Number` | `1.5`     | Altitude of camera in screen units                         |
+| `width`       | `Number` | `1`       | Width of viewport |
+| `height`      | `Number` | `1`       | Height of viewport |
+| `latitude`    | `Number` | `0`      | Latitude of viewport center  |
+| `longitude`   | `Number` | `0`    | Longitude of viewport center  |
+| `zoom`        | `Number` | `11`      | Map zoom (scale is calculated as `2^zoom`) |
+| `pitch`       | `Number` | `0`       | The pitch (tilt) of the map from the screen, in degrees (0 is straight down) |
+| `bearing`     | `Number` | `0`       | The bearing (rotation) of the map from north, in degrees counter-clockwise (0 means north is up) |
+| `altitude`    | `Number` | `1.5`     | Altitude of camera in screen units  |
 
 
 Remarks:
  - Altitude has a default value that matches assumptions in mapbox-gl
  - `width` and `height` are forced to 1 if supplied as 0, to avoid
    division by zero. This is intended to reduce the burden of apps to
    to check values before instantiating a `Viewport`.
- -  When using mercatorProjection, per cartographic tradition, longitudes and
+ -  When using Mercator projection, per cartographic tradition, longitudes and
    latitudes are specified as degrees.
 
 
-### `project(lngLatZ, opts)`
+## Methods
+
+##### `project(lngLatZ, opts)`
```

Comment:
Nit: 

I feel that this "5 hash" subtitle formatting (introduced to get sub headers to look a little better when read directly on github) is mainly used by deck.gl at this stage.

I have consistently use "3 hash" subtitles for a long time. And (although it is currently plagued by header styling issues) `gatsby-theme-ocular` header styles should be adapted to make those look good in generated web sites. 

I feel it is also more logical to use 3 hashes as it represents one indentation per level.

Not a big issue of course, this should only lead to a minor styling difference.